dipy/reconst/dki.py

```diff
@@ -1799,6 +1799,7 @@ def __init__(self, gtab, *args, fit_method="WLS", return_S0_hat=False, **kwargs)
             if isinstance(self.convexity_level, str):
                 if self.convexity_level == "full":
                     self.sdp_constraints = load_sdp_constraints("dki")
+                    # NOTE: load my constraint matrices instead, see if they work!
```

Comment:
Not sure to understand this comment. 

it seems that we always use your constraint matrices by default. Right ?

maybe comment to remove

Reply:
yes you're right, needs to be removed, I will do it